docs/api/proxy-based-tracking.md

```diff
@@ -0,0 +1,145 @@
+---
+id: proxy-based-tracking
+title: Proxy-based Tracking
+sidebar_label: Proxy-based Tracking
+hide_title: true
+---
+
+# Proxy-based Tracking
+
+This document describes about `useTrackedState` hook.
+
+## How does this get used?
+
+`useTrackedState` is a hook that can be used instead of `useSelector`.
```

Comment:
I'd like to rephrase this intro paragraph.  Something like:

> `useTrackedState` allows components to read values from the Redux store state.  It is similar to `useSelector`, but uses an internal tracking system to detect which state values are read in a component, without needing to define a selector function.

Reply:
Let me copy&paste. 😄